public/scss/home.scss

```diff
@@ -0,0 +1,73 @@
+.tx-bar {
+  border-width: 4px;
+  border-radius: 6px;
+}
+
+.tx-bar.tx-regular {
+  border-left-style: solid;
+  border-color: $regular-light;
+}
+
+.tx-bar.tx-vote {
+  border-right-style: solid;
+  border-color: $vote-light;
+}
+
+.tx-bar.tx-ticket {
+  border-left-style: solid;
+  border-color: $ticket-light;
+}
+
+.tx-bar.tx-rev {
+  border-right-style: solid;
+  border-color: $rev-light;
+}
+
+.tx-gauge {
+  height: 4px;
+  margin: 0 1px;
+  width: 25%;
+  transition: width 1s ease-in-out;
+}
+
+.tx-gauge.tx-regular {
+  background-color: $regular-light;
+}
+
+.tx-gauge.tx-vote {
+  background-color: $vote-light;
+}
+
+.tx-gauge.tx-ticket {
+  background-color: $ticket-light;
+}
+
+.tx-gauge.tx-rev {
+  background-color: $rev-light;
+}
+
+.mempool-card {
+  background-color: #e3eaed;
+}
+
+.darkBG .mempool-card {
+  background-color: #1c1512;
+}
+
+.block-card {
+  background-color: #efece8;
```

Comment:
Not sure I love this peach/rust look to the Latest Blocks card.  I don't feel strongly about it though.